include/openssl/evp.h

```diff
@@ -928,6 +928,45 @@ OPENSSL_EXPORT EVP_PKEY *EVP_PKEY_kem_new_raw_key(int nid,
 // to the secret key in |key|.
 OPENSSL_EXPORT int EVP_PKEY_kem_check_key(EVP_PKEY *key);
 
+// ASN1 functions
+
+typedef struct evp_pkey_asn1_method_st EVP_PKEY_ASN1_METHOD;
```

Comment:
nit: should we move this to base.h with [the other `evp_pkey_*_st` typedefs](https://github.com/aws/aws-lc/blob/c28d0cb27759b09eb305a98f822c0223eeb2c727/include/openssl/base.h#L345)?